src/linear_algebra/alternating.lean

```diff
@@ -344,3 +375,25 @@ begin
 end
 
 end alternating_map
+
+namespace linear_map
+
+variables {N'₂ : Type*} [add_comm_group N'₂] [semimodule R N'₂] [fintype ι]
+
+/-- Composition with a linear map before and after alternatization are equivalent. -/
+lemma comp_multilinear_map_alternatization (g : N' →ₗ[R] N'₂)
+  (f : multilinear_map R (λ _ : ι, M) N') :
+  (g.comp_multilinear_map f).alternatization = g.comp_alternating_map (f.alternatization) :=
+begin
+  ext,
+  simp [multilinear_map.alternatization_apply],
+  congr,
+  ext1 σ,
+  -- `linear_map.map_smul` doesn't work here, as there is no ℤ-module structure to use
```

Comment:
#5430 fixes this problem, it may make sense to review that first

Reply:
I take that back, let's put this in first so that I can use it to justify the other PR.

Reply:
Do I understand correctly: if we merge #5430, this whole proof could be `by { ext , simp [multilinear_map.alternatization_apply, linear_map.map_smul], }`? If that's the case, I'm happy with merging as-is and cleaning up in #5430. Otherwise, please see the suggestions.

Reply:
I don't know for sure - I'll make the cleanup you suggest just in case #5430 helps less than I expect.

Reply:
Your conjecture was correct - with that PR, this is simply `by { ext, simp [multilinear_map.alternatization_apply] }`